src/main.cpp

```diff
@@ -4850,61 +4862,68 @@ void static ProcessGetData(CNode* pfrom, const Consensus::Params& consensusParam
                 // Pruned nodes may have deleted the block, so check whether
                 // it's available before trying to send.
                 if (send && (mi->second->nStatus & BLOCK_HAVE_DATA))
-                {
+                  {
                     // Send block from disk
                     CBlock block;
                     if (!ReadBlockFromDisk(block, (*mi).second, consensusParams))
-                        assert(!"cannot load block from disk");
-                    if (inv.type == MSG_BLOCK)
-		      {
-    	                pfrom->blocksSent += 1;
-                        pfrom->PushMessage(NetMsgType::BLOCK, block);
-		      }
-
-                    // BUIP010 Xtreme Thinblocks: begin section
-                    else if (inv.type == MSG_THINBLOCK || inv.type == MSG_XTHINBLOCK) {
-                        LogPrint("thin", "Sending xthin by INV queue getdata message\n");
-                        SendXThinBlock(block, pfrom, inv);
-                    }
-                    // BUIP010 Xtreme Thinblocks: end section
-
-                    else // MSG_FILTERED_BLOCK)
-                    {
-                        LOCK(pfrom->cs_filter);
-                        if (pfrom->pfilter)
-                        {
-                            CMerkleBlock merkleBlock(block, *pfrom->pfilter);
-                            pfrom->PushMessage(NetMsgType::MERKLEBLOCK, merkleBlock);
+                      {
+                        // its possible that I know about it but haven't stored it yet
+                        LogPrint("thin", "unable to load block %s from disk\n", (*mi).second->phashBlock ? (*mi).second->phashBlock->ToString() : "");
+                        // no response
```

Comment:
since the code was indented though, I'll also take this opportunity to switch the parens to BSD